doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -90,11 +91,24 @@ Rewriting with Leibniz and setoid equality
      any of them can be rewritten.  If not specified, only the first occurrence
      in the conclusion is replaced.
 
-     If :n:`at @occs_nums` is specified, rewriting is always done with
-     :ref:`setoid rewriting <generalizedrewriting>`, even for Leibniz’s equality,
-     which means that you must `Require Setoid` to use that form.
-     However, note that :tacn:`rewrite` (even when using setoid rewriting) and
-     :tacn:`setoid_rewrite` don't behave identically (as already mentioned above).
+     .. note::
+
+        If :n:`at @occs_nums` is specified, rewriting is always done
+        with :ref:`setoid rewriting <generalizedrewriting>`, even for
+        :term:`Leibniz equality`, which means that you must `Require
+        Setoid` to use that form.  However, note that :tacn:`rewrite`
```

Comment:
Is that still true? I think it has been several years since `Require Setoid` is no longer required (fortunately).

Reply:
Unfortunately yes, this is still true. And if you didn't notice it, it's probably because you load it transitively. E.g. by doing `Require Import Arith.`

Reply:
Unbelievable...